nntrainer/tensor/hgemm/hgemm.cpp

```diff
@@ -27,7 +27,9 @@
 void hgemm_noTrans(const __fp16 *A, const __fp16 *B, float *C32, unsigned int M,
                    unsigned int N, unsigned int K, float alpha, float beta) {
   if (alpha == 1.F && beta == 0.F) {
-    if (M % 8 == 0 && N % 8 == 0 && K % 8 == 0) {
+    if (M % 8 == 0 && N % 16 == 0 && K % 8 == 0) {
```

Comment:
Can’t you do it with bitwise operations?
I know that bit operations are generally faster, but I don't know if there is actually a performance difference after optimization.
If so, I think we can improve our performance by changing it.